src/main/steps/index.test.ts

```diff
@@ -161,18 +161,25 @@ describe('Steps', () => {
 
   describe('isConditionalOrderReadyToSubmit()', () => {
     it('returns false if nextStepUrl is /continue-with-your-application', () => {
-      const isApplicationReadyToSubmitBoolean = isConditionalOrderReadyToSubmit(CONTINUE_WITH_YOUR_APPLICATION);
+      const isApplicationReadyToSubmitBoolean = isConditionalOrderReadyToSubmit(
+        CONTINUE_WITH_YOUR_APPLICATION,
+        HOME_URL
+      );
       expect(isApplicationReadyToSubmitBoolean).toBeFalsy();
     });
 
     it('returns true if nextStepUrl is /', () => {
-      const isApplicationReadyToSubmitBoolean = isConditionalOrderReadyToSubmit(HOME_URL);
+      const isApplicationReadyToSubmitBoolean = isConditionalOrderReadyToSubmit(
+        HOME_URL,
+        CHECK_CONDITIONAL_ORDER_ANSWERS_URL
+      );
       expect(isApplicationReadyToSubmitBoolean).toBeTruthy();
     });
 
     it('returns true if nextStepUrl contains /check-your-conditional-order-answers', () => {
       const isApplicationReadyToSubmitBoolean = isConditionalOrderReadyToSubmit(
-        `${CHECK_CONDITIONAL_ORDER_ANSWERS_URL}?lng=eng`
+        `${CHECK_CONDITIONAL_ORDER_ANSWERS_URL}?lng=eng`,
+        `${CHECK_CONDITIONAL_ORDER_ANSWERS_URL}`
```

Comment:
for these tests, the second argument should be the applicantApplyForConditionalOrder field, so either a "Yes", or "No" or undefined field

Reply:
Ok, I'll make that change.